File: android-sdk/src/main/java/com/checkout/android_sdk/Models/PhoneModel.java
package com.checkout.android_sdk.Models;

/**
 * Http request Phone object model
 */
public class PhoneModel {

    private String countryCode;
    private String number;

    public PhoneModel(String countryCode, String number) {
        this.countryCode = countryCode;
        this.number = number;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public String getNumber() {
        return number;
    }
}

File: android-sdk/src/main/java/com/checkout/android_sdk/Models/PhoneModel.kt
package com.checkout.android_sdk.Models

import com.google.gson.annotations.SerializedName

/**
 * Http request Phone object model
 */
class PhoneModel(
        @SerializedName("country_code")
        val country_code: String,
        @SerializedName("number")
        val number: String
)
